.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,33 @@
+favicon.ico,1560689577946,2e1becf7e21a9cf9ef3c7f3bc84ec1c5a51a74e4240c0910b326a69c8859d5a1
+manifest.json,1560689577936,7b6115f36048ec652ab2d945dd11769861949f830e8f776dbc5de9cd1074b27b
+precache-manifest.45fb4bfc0cbb51de248bf03774ac3b10.js,1560689577947,fa7158e6957448dfdabb9d6b832b8357015a3fa1db3ce4887444453c367cca63
+mstile-150x150.png,1560689577942,725c62857026c00f11c76dac219a673596a691a9d4c6b472d17d9bb1fc178a94
+robots.txt,1560689577947,8c8992414cc7101f8c9952a566181d667e7343481f8e7b05c55103b1bfa0d899
+safari-pinned-tab.svg,1560689577952,5f273bb02fc23500b6223a4c45aaade132255bcf532aff01218cc8bef3ba3ecb
+service-worker.js,1560689577954,fb720d4155e9dcb63119261a3abbe010996a6551cd37e393452e7da8a99d9a29
+css/navSignManifest.616b3d8c.css,1560689578016,d58cb01a583d2c0760de3c4e4c78a57acf006c2c7a6dd7945276ee31e7c53a4b
+img/canarilogo.a10313d7.png,1560689577944,cc7a6a3693f351df24058927272d803521bef1b3e7e0d2c97cd9c1ebd4e50d51
+css/login~navSignManifest~register.a6cefaa6.css,1560689578003,8ada9b7abb3a9aa807ff6c66333d2802f6845998f08ac6bb2887eff2ba29d9ec
+js/app.2eb64905.js,1560689577987,eb0303fa0dd11a181cedadfe5bd67d720aa6bd6b1656d57f6f10d122d234f891
+js/controlPanel.3825b0c2.js,1560689578004,cd9c227a0a3316929babf07fa58d671fa96a0813715eb169dfe450abf8d3f873
+js/controlPanel.3825b0c2.js.map,1560689578052,0a137df949b53060b9884540821c55ed48297a2e965e88f98422d6f62909bbf7
+js/login.fa8e2797.js,1560689578004,2c7f4e310165ff408b87030214c424acb10500333cd49ea93b4b2ddf8cfc295e
+js/app.2eb64905.js.map,1560689578032,4aa826c4db8ea6f5fe61c824358e26ab09cff55673adf34c19508982f038d538
+js/login.fa8e2797.js.map,1560689578073,03c8dfac6eb4240b53153c483b4c70fb197db9de19927d8cb41788719652d7cf
+js/login~navSignManifest~register.4308cc1b.js,1560689578048,ab1c02f54e4f00a9b934447dfec14d99ca41acce8dc4ca04d750249b742f9632
+js/logOutBtn.7dfdbc6f.js,1560689577980,fc413fc237040666d1453615c2aa0e9ad7e20d62667fffc1536c2d10d0a51ae4
+js/logOutBtn.7dfdbc6f.js.map,1560689578067,cd860e6182eb0f9fbf3479d7ee940e27cd343d4fb39380a35e9d81846aaeff86
+js/navSignManifest.10d6dd36.js,1560689578058,aa5dc130ac915d64bffec26b7599097dc4cf48768bf87cc607119ab6733991b3
+js/register.9e99284d.js,1560689578020,f59d4358a677c8285b8a312de2f9ec9601f2d0597d08ac6a8a9fa8fe453a3c26
+js/register.9e99284d.js.map,1560689578091,9a0e1931e476ac8125640c9296ec0913fef4894e8e1a80dc1d54e64acb9b7d0a
+css/chunk-vendors.89bf4e3b.css,1560689577966,0b1a999235efebc9f2a4a9c73905ef19b13fc715e4a13611fa9ccd07ec4b55ef
+js/navSignManifest.10d6dd36.js.map,1560689578077,d2f42911f65c3c456067cce20e9df24fe0b085e481d782e6650212474d17200b
+js/login~navSignManifest~register.4308cc1b.js.map,1560689578072,77e19f68cacd709ecb00765d80044e80e2375cfa658734f36892f37496c7569a
+js/chunk-vendors.5f39dec5.js,1560689578286,e80903aef2c4d2faa4d1455c39bfe999c8cf17cc568953ee0b061303410cae13
+js/chunk-vendors.5f39dec5.js.map,1560689578168,fd27ade2269d0e2ef2b28c2f2aa33d7720fed650211d057900882290ed8a2bda
```

Comment:
Is checking in this file a requirement?

Reply:
No, it's completely dispensable, in fact the whole /dist folder is, since that folder is generated when the project is compiled, and it will always change the cache serial.